### ISSUE TITLE
Got the error "value must be string" 

### ISSUE THREAD

**Issue by Speira** [jodit/jodit-react#110]
I dit this function:
```js 
function Wysiwyg() {
  const editor = React.useRef(null)
  const [content, setContent] = React.useState('')
  const config = {
    readonly: false, // all options from https://xdsoft.net/jodit/doc/
  }

  return (
    <JoditEditor
      ref={editor}
      value={content}
      config={config}
      // eslint-disable-next-line
      tabIndex={1} // tabIndex of textarea
      onBlur={(newContent) => setContent(newContent)}
      onChange={() => {}}
    />
  )
}
```
But when I click on the are I get an error message "value must be string".
I use React 16.13

**Comment by Saff-Elli-Khan** [comment]
Same error on my application!!

**Comment by pedroraimondi** [comment]
I have the same problem. Did anyone fix it??

**Comment by OL-gag** [comment]
same here, with a blank page when leaving the textarea

**Comment by Fedeorlandau** [comment]
Same here, using     
"jodit-react": "^1.0.66",
 "react": ">=16.8.6",

**Comment by OL-gag** [comment]
I've just removed the ref={editor} line, seems fixe my issue. Don't know if is usefull...

**Comment by Kristofferkirk** [comment]
OnBlur currently return a "FocusEvent" and not a string value. Which is why the error occurs.
I managed to get the value by creating a OnBlur handler like this:

`
const onBlur = (event: any) => {
    const editorValue = event.target.innerHTML;
    setContent(editorValue);
  };
`

Then props:
`OnBlur={onBlur}`

**Comment by Kristofferkirk** [comment]
Duplicate issue
https://github.com/jodit/jodit-react/issues/93

**Comment by azw2** [comment]
For my case i had to downgrade react-jodit to version 1.0.51.

**Comment by Octopusboom** [comment]
> OnBlur currently return a _FocusEvent_ and not a string value. Which is why the error occurs.
> I managed to get the value by creating a OnBlur handler like this:
> 
> `const onBlur = (event: any) => { const editorValue = event.target.innerHTML; setContent(editorValue); };`
> 
> Then props:
> `OnBlur={onBlur}`
> 
> _Workaround, but either the docs/types should be updated or make the return value a string value again._

yes it solves part of the issue, but now evetime you click a button from the editor, you have to click twice.

**Comment by 4eshircat** [comment]
Hi guys!
I dealt with this problem like this:
`onBlur={newContent => setContent(newContent.target.innerHTML)}`
And everything works perfectly 👏🏻

**Comment by fukszssl247** [comment]
I did some digging seems this bug was introduced in 10.0.54 because 10.0.53 seems to wok just fine.

https://github.com/jodit/jodit-react/compare/1.0.53...1.0.54
